webinterface/pages/DDA_Quant.py

```diff
@@ -90,14 +90,13 @@ def _main_page(self):
 
             with st.expander("Additional parameters"):
 
-                with open("webinterface/configuration/dda_quant.json") as file:
+                with open("../webinterface/configuration/dda_quant.json") as file:
```

Comment:
I could not test this at the moment to confirm that the change was needed, but I trust that it was justifiable.